### PR TITLE
New reptile sounds

### PR DESCRIPTION
**Pull request by Minemoder5000**
## About the PR

Reptilians and Kobolds can now thump their tail.

## Why / Balance

Additional sounds for roleplay purposes. Tail thump also included in SS13 Shiptest

## Technical details

Added new type ReptilianBodyEmotes to speech_emote_sounds.yml
Added audio paths in speech_emote_sounds.yml
Added relevant emotes to emotes.ftl
Gave allowedEmotes component to Reptilians and Kobolds
Added new Emote icons
## Media

https://github.com/user-attachments/assets/83c9993f-ca48-4005-aeee-9a19d64c1255

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: Minemoder, ArtisticRoomba, Sarahon
- add: Reptiles and Kobolds can now thump their tail.

### CODE REVIEW

**Comment by ArtisticRoomba** [comment]
Here's what the sounds sound like.

https://github.com/user-attachments/assets/ec14a248-0016-43fb-95c4-d68b5e032ae7

File: BuildChecker/BuildChecker.csproj
Comment (VasilisThePikachu) [inline BuildChecker/BuildChecker.csproj:1]: Revert changes to BuildChecker.csproj

**Review by VasilisThePikachu — DISMISSED** [review]

File: BuildChecker/BuildChecker.csproj
Comment (ArtisticRoomba) [reply BuildChecker/BuildChecker.csproj:1]: Hehe forgor
Was teaching him git

**Comment by VasilisThePikachu** [comment]
undraft when you are ready!

**Review by UbaserB — CHANGES_REQUESTED** [review]
Can you make the huff and hiss less guttural and more high pitched sorta (to match the other lizard emotes)?

**Comment by Minemoder5000** [comment]
Sure, I'll see if I can edit the current huff or find a different one that fits better

**Comment by foob1** [comment]
I suggest you read through https://github.com/space-wizards/space-station-14/pull/29448 since there was a lot of feedback on lizard noises there. I'd also like to add my own comments here but I already discussed it in the PR I linked

**Comment by UbaserB** [comment]
> Sure, I'll see if I can edit the current huff or find a different one that fits better

Any progress on that?

**Comment by Minemoder5000** [comment]
No, not really. I've been struggling to find some sounds that work and fit with current reptilian sounds. What parts of the hiss and huff were too guttural? I think I have a good idea of why the huff won't fit but I just want a bit more info that I can work with to find or make some good sounds.

**Comment by UbaserB** [comment]
> No, not really. I've been struggling to find some sounds that work and fit with current reptilian sounds. What parts of the hiss and huff were too guttural? I think I have a good idea of why the huff won't fit but I just want a bit more info that I can work with to find or make some good sounds.

It's like, too low pitched with deep vibrations if you sorta get what i mean.

**Comment by Minemoder5000** [comment]
Does that also apply to the hiss? Because the only low pitched rumbly part at the start of the sound isn't really that low pitched. I could trim that part of the hiss if it'd make the sound work.

**Comment by Minemoder5000** [comment]
Alright so instead of letting this sit here forever, I've decided to just trim this PR down to the tail thump and not add the huff or hiss. How should I go about doing that? Should I remove the yaml changes, or comment them out so someone in the future can take it over? Should I remove or keep the huff and hiss emote icons for future use?

**Comment by UbaserB** [comment]
It'd be accepted then yeah i think

**Review by UbaserB — APPROVED** [review]

File: Resources/Audio/Voice/Reptilian/attritbutions.yml [hunk 1]
```diff
@@ -2,3 +2,8 @@
   copyright: '"scream_lizard.ogg" by Skyrat-SS13'
   license:
   source: https://github.com/Skyrat-SS13/Skyrat-tg/pull/892
+
+- files: [reptilian_tailthump.ogg]
+  copyright: "Taken from https://github.com/shiptest-ss13/Shiptest/pull/2159/commits/034844b6946d15a680bf8f9edab2ded7051b305c#diff-5c6dee940bb4ac50e1bb77b17f6a84a083f901a4289823c40852520e8d016edc"
+  license: "CC-BY-SA-3.0"
```
Comment (slarticodefast) [inline Resources/Audio/Voice/Reptilian/attritbutions.yml:8]: This should link to the original file on freesound and have the correct licence
https://freesound.org/people/TylerAM/sounds/389665/

**Review by slarticodefast — CHANGES_REQUESTED** [review]
The tailslap.png image is missing a licence and attribution in Resources/Textures/Interface/Emotes/attributions.yml
Otherwise this is good to merge.

File: Resources/Prototypes/Voice/speech_emotes.yml [hunk 2]
```diff
@@ -242,6 +242,31 @@
     - snapping fingers
     - snapped fingers
 
+- type: emote
+  id: Thump
+  name: chat-emote-name-thump
+  category: Hands
```
Comment (beck-thompson) [inline Resources/Prototypes/Voice/speech_emotes.yml:248]: Hey you might want to make new category! If its hands, stuff like #32588 might block it even if it shouldn't (E.g cuffs would block this even though that doesn't make sense.

Comment (Minemoder5000) [reply Resources/Prototypes/Voice/speech_emotes.yml:248]: I'll worry about this if it gets merged

**Review by slarticodefast — APPROVED** [review]

**Comment by slarticodefast** [comment]
Random, unrelated test fail.
Code looks good to me. Thank you for your contribution!

**Pull request merged**